apps/meteor/ee/app/livechat-enterprise/server/api/units.ts

```diff
@@ -1,55 +1,10 @@
 import { API } from '../../../../../app/api/server';
-import { deprecationWarning } from '../../../../../app/api/server/helpers/deprecationWarning';
 import { findUnits, findUnitById, findUnitMonitors } from './lib/units';
 import { LivechatEnterprise } from '../lib/LivechatEnterprise';
+import { findAllDepartmentsAvailable, findAllDepartmentsByUnit } from '../lib/Department';
 
 API.v1.addRoute(
-	'livechat/units.list',
```

Comment:
We can cleanup this definition now -> https://github.com/RocketChat/Rocket.Chat/blob/899b95f3141d47487995607088cde9b8422296ce/apps/meteor/ee/definition/rest/v1/omnichannel/businessUnits.ts#L7